Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -1448,12 +1448,37 @@ public static bool Install(NugetPackage package, bool refreshAssets = true)
                 {
                     EditorUtility.DisplayProgressBar(string.Format("Installing {0} {1}", package.Id, package.Version), "Importing Package", 0.95f);
                     AssetDatabase.Refresh();
+                    ModifyImportSettingsIfRoslynAnalyzer(package.Id, package.Version);
                     EditorUtility.ClearProgressBar();
                 }
             }
             return installSuccess;
         }
 
+        private static void ModifyImportSettingsIfRoslynAnalyzer(string id, string version)
+        {
+            var dir = $"Assets/Packages/{id}.{version}";
```

Comment:
You should use `NugetConfigFile.RepositoryPath` instead of the hardcoded path

Reply:
Ah but you do need relative paths :thinking:, could take the same approach as used when adding support for [native files](https://github.com/GlitchEnzo/NuGetForUnity/pull/421/files#diff-460028a9a166ac23a7e76cf1282330ca51138fc1d7557336de43c60183de8f44R1393)